### ISSUE TITLE
Join function can be removed

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#585]
We're no longer joining arrays using commas, so the function defined in section 3.3 of each techniques document is no longer being called. It should be possible to delete the section from each document.

**Comment by gregoriopellegrino** [comment]
Done in ONIX.

As soon as EPUB Metadata is merged, we can close this issue.
